layers/meta-resin-edison/recipes-core/images/resin-image-initramfs.bbappend

```diff
@@ -0,0 +1 @@
+IMAGE_ROOTFS_MAXSIZE = "16384"
```

Comment:
This looks very strange. A p7 partition of 20 megs? The idea is that we need to make sure we don't break resinhup.